### PR TITLE
ML - New UI component for icon, title and description of an item

### PR DESCRIPTION
**Pull request by llali**
This PR fixes #

### CODE REVIEW

File: extensions/machine-learning/src/views/models/azureModelsComponent.ts [hunk 1]
```diff
@@ -49,9 +52,32 @@ export class AzureModelsComponent extends ModelViewBase implements IDataComponen
 			this._downloadedFile = undefined;
 		});
 
+		this._emptyModelsComponent = new DataInfoComponent(this._apiWrapper, this);
+		this._emptyModelsComponent.width = 300;
+		this._emptyModelsComponent.height = 250;
+		this._emptyModelsComponent.iconSettings = {
+			css: { 'padding-top': '20px' },
+			width: 100,
```
Comment (halerankin) [inline extensions/machine-learning/src/views/models/azureModelsComponent.ts:60]: The empty state icon (documents and magnifying glass) is 128x128.

Comment (llali) [reply extensions/machine-learning/src/views/models/azureModelsComponent.ts:60]: should I change the height to 128?

File: extensions/machine-learning/src/views/models/azureResourceFilterComponent.ts [hunk 2]
```diff
@@ -142,6 +123,7 @@ export class AzureResourceFilterComponent extends ModelViewBase implements IData
 		this._azureAccounts = await this.listAzureAccounts();
 		if (this._azureAccounts && this._azureAccounts.length > 0) {
 			let values = this._azureAccounts.map(a => { return { displayName: a.displayInfo.displayName, name: a.key.accountId }; });
+			//values = [{ displayName: 'Select account ...', name: '' }].push(values);
```
Comment (halerankin) [inline extensions/machine-learning/src/views/models/azureResourceFilterComponent.ts:126]: Will this commented code be removed?

Comment (llali) [reply extensions/machine-learning/src/views/models/azureResourceFilterComponent.ts:126]: yes I'll remove it

**Review by halerankin — APPROVED** [review]

**Pull request merged**